content/docs/15.how-to-guides/azure-workload-id.md

```diff
@@ -0,0 +1,214 @@
+
+```shell
+OBJECT_ID=$(az identity show --name $ID_NAME --resource-group $RESOURCE_GROUP --query 'principalId' --output tsv)
+MANAGED_CLIENT_ID=$(az identity show --name $ID_NAME --resource-group $RESOURCE_GROUP --query clientId --output tsv)
+AKS_OIDC_ISSUER="$(az aks show --name "${AKS_NAME}" --resource-group "${RESOURCE_GROUP}" --query "oidcIssuerProfile.issuerUrl" --output tsv)"
+```
+
+## Link Identity Resources
+
+One of the more challenging aspects of this setup is correctly linking together the various resources. This section covers how to tie the managed identities to the resources to allow access by the Kestra application. 
+
+### Create role assignment for created user 
+
+This is one of the more critical steps as it sets the permission the resource has on the Key Vault. As Kestra needs to read and write secrets to the vault, the "Key Vault Secrets Officer" provides least priviledged access for this operation. Further details on this role can be found [here](https://learn.microsoft.com/en-us/azure/key-vault/general/rbac-guide?tabs=azure-cli#azure-built-in-roles-for-key-vault-data-plane-operations). 
+
+```shell
+az role assignment create \
+  --assignee-object-id $OBJECT_ID \
+  --role "Key Vault Secrets Officer" \
+  --scope $(az keyvault show --name $KEYVAULT_NAME --query id -o tsv)
+```
+
+
+### Create the service account in the AKS Cluster
+
+First, we must switch context to the newly created cluster
+
+```shell
+az aks get-credentials --resource-group $RESOURCE_GROUP --name $AKS_NAME
+```
+
+Next, create service account in the same namespace as where you wish to deploy Kestra. 
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    azure.workload.identity/client-id: "${MANAGED_CLIENT_ID}"
+  name: "${SERVICE_ACCOUNT_NAME}"
+  namespace: "${SERVICE_ACCOUNT_NAMESPACE}"
+EOF
+```
+
+Finally, we need to link this service account to the OIDC issuer associated with our managed identity. We do this by creating the following federeated credential. 
+
+```shell
+az identity federated-credential create \
+  --name ${FEDERATED_IDENTITY_CREDENTIAL_NAME} \
+  --identity-name "${ID_NAME}" \
+  --resource-group "${RESOURCE_GROUP}" \
+  --issuer "${AKS_OIDC_ISSUER}" \
+  --subject system:serviceaccount:"${SERVICE_ACCOUNT_NAMESPACE}":"${SERVICE_ACCOUNT_NAME}" \
+  --audience api://AzureADTokenExchange
+```
+
+## Deploying Kestra
+
+Before we deploy Kestra, we need to modify the `values.yaml` of the helm chart using some of the information above. 
+
+### Configure Secrets Manager
+
+```yaml
+configuration:
+  kestra:
+    secret:
+      type: azure-key-vault
+      azure-key-vault:
+        vaultName: ${KEYVAULT_NAME}
+        workloadIdentityClientId: ${MANAGED_CLIENT_ID}
+```
+
+### Service Account
+
+Simply add the service account name in the file. There is already a value present so overwrite with your value defined above. 
```

Comment:
```suggestion
Make sure to add the service account name to the `values.yaml` file. Since a value is already present, overwrite it with your value defined above. 
```